Bottom: 79807f9136b711f1415bacf051148cac6fa28ff2
Top:    2898bcf1cacb7384188cb1914ad7feb2316b927b
Author: Ian Kent <raven@themaw.net>
Date:   2013-05-20 23:48:44 +0800

Refresh of autofs-5.0.7-add-host-map-reload-option.patch

---

diff --git a/man/auto.master.5.in b/man/auto.master.5.in
index 7907ffa..58bd510 100644
--- a/man/auto.master.5.in
+++ b/man/auto.master.5.in
@@ -253,7 +253,10 @@ which corresponds to a hostname will allow access to the exports of that
 host. The hosts map cannot be dynamically updated and requires a HUP signal
 to be sent to the daemon for it to check hosts for an update. Due to possible
 hierarchic dependencies within a mount tree, it might not be completely
-updated during the HUP signal processing.
+updated during the HUP signal processing. It is also possible that dependencies
+can cause the hosts map to become inoperable so a configuration entry,
+HOSTS_MAP_RELOAD, determines whether to the map is refreshed upon receiving the
+signal. The default is, "no".
 .P
 For example, with an entry in the master map of
 .nh
diff --git a/redhat/autofs.sysconfig.in b/redhat/autofs.sysconfig.in
index a8992c4..3d481e7 100644
--- a/redhat/autofs.sysconfig.in
+++ b/redhat/autofs.sysconfig.in
@@ -48,6 +48,15 @@ MOUNT_NFS_DEFAULT_PROTOCOL=4
 #
 #LOGGING="none"
 #
+# HOSTS_MAP_RELOAD - try and refresh the exports of the internal
+#                    hosts map when a HUP signal is received. Note
+#                    there can easily be unresolvable dependencies
+#                    when changing the exported mount tree on servers
+#                    so this configuration entry is set to "no" by
+#                    default.
+#
+#HOSTS_MAP_RELOAD="no"
+#
 # Define base dn for map dn lookup.
 #
 # Define server URIs
diff --git a/samples/autofs.conf.default.in b/samples/autofs.conf.default.in
index 1da89cf..77a387a 100644
--- a/samples/autofs.conf.default.in
+++ b/samples/autofs.conf.default.in
@@ -47,6 +47,15 @@ BROWSE_MODE="no"
 #
 #LOGGING="none"
 #
+# HOSTS_MAP_RELOAD - try and refresh the exports of the internal
+#                    hosts map when a HUP signal is received. Note
+#                    there can easily be unresolvable dependencies
+#                    when changing the exported mount tree on servers
+#                    so this configuration entry is set to "no" by
+#                    default.
+#
+#HOSTS_MAP_RELOAD="no"
+#
 # Define server URIs
 #
 # LDAP_URI - space seperated list of server uris of the form
